firefox-ios/Storage/Rust/RustAutofill.swift

```diff
@@ -108,11 +107,14 @@ public class RustAutofill {
                 completion(nil, error)
                 return
             }
-            do {
-                let id = try self.storage?.addCreditCard(cc: creditCard.toUpdatableCreditCardFields())
-                completion(id!, nil)
-            } catch let err as NSError {
-                completion(nil, err)
+
+            self.encryptCreditCard(creditCard: creditCard) { encCreditCard in
+                do {
+                    let id = try self.storage?.addCreditCard(cc: encCreditCard)
+                    completion(id!, nil)
```

Comment:
Nit: we might consider using `guard let id = ` here and if that fails to throw an error from this do/catch block, rather than force-unwrap the `id!`